### PR TITLE
Add files via upload- Added thrown together streak counter.

### PR DESCRIPTION
**Pull request by adamthedog**
Added streak counter. I didn't realize that stopping on a right guess keeps it going, but I'm waay too tired to fix it now.
If someone could for me, that's be great...

### CODE REVIEW

File: neuralNetworkLanguageDetection.pde [hunk 1]
```diff
@@ -1,214 +1,225 @@
...
+int LANGUAGE_COUNT = 13;
+int MIDDLE_LAYER_NEURON_COUNT = 19;
+String[][] trainingData = new String[LANGUAGE_COUNT][];
+int SAMPLE_LENGTH = 15;
+int INPUTS_PER_CHAR = 27;
+int INPUT_LAYER_HEIGHT = INPUTS_PER_CHAR*SAMPLE_LENGTH+1;
+int OUTPUT_LAYER_HEIGHT = LANGUAGE_COUNT+1;
+int lineAt = 0;
+int iteration = 0;
+int streak;
```
Comment (adamthedog) [inline neuralNetworkLanguageDetection.pde]: New streak variable

**Comment by hanss314** [comment]
Put the code that increments the counter in the train() method. Actually, I'm going to do that.

**Comment by phase** [comment]
You did something wrong with the line endings. Make sure yours are the same as the source file's.

**Comment by ZweiSpeedruns** [comment]
To elaborate, I believe cary did make this in windows, which uses different line ending than linux, and I know that adamthedog runs ubuntu.

**Comment by hanss314** [comment]
I don't see anything wrong with the line endings.

**Comment by adamthedog** [comment]
Hanss314, if you are going to add it, should I close this pull and you make a new one? I'm working on an anti-aliasing feature for all of Cary's projects, so I could just paste my code in if yours gets pushed. *Ahem, Cary*

**Comment by hanss314** [comment]
Yeah sure, you could close yours. I'm pretty sure you can also make prs to my branch. Just replace the username in the url with my username to see my branch.

**Comment by hanss314** [comment]
I just pushed a new commit to my branch containing your changes and a few more. If you want you can close your branch and push commits to my branch. I'll also try to get cary on discord.

**Pull request closed without merging**